src/css/legal.scss

```diff
@@ -6,41 +6,70 @@
         text-decoration: none;
     }
 
+    h1 {
+        font-size: 48px;
+        line-height: 52px;
+        letter-spacing: -2px;
+        font-weight: 800;
+    }
+
+    h2 {
+        font-family: 'Work Sans', sans-serif;
+        font-size: 32px;
+        line-height: 40px;
+        font-weight: 700;
+        letter-spacing: -1px;
+        margin: 3rem 0 1rem;
+        &:first-child {
+            margin: 0 0 2rem;
+        }
+    }
+
+    h3 {
+        font-family: 'Work Sans', sans-serif;
+        font-size: 24px;
+        font-weight: 500;
+        line-height: 30px;
+        letter-spacing: -.5px;
+        margin: 2rem 0 1.5rem;
+    }
+
+    h4 {
+        font-family: 'Work Sans', sans-serif;
+        font-size: 16px;
+        font-weight: bold;
+        line-height: 25px;
+        letter-spacing: -.5px;
+        margin: 1rem 0;
+    }
```

Comment:
Not necessarily for right this moment, but if these are all design system settings I wonder if we should look into pulling them to `typography.scss`. At a glance, they're different values, which is interesting---would love to dig in a bit and see if the issue is that the design system is oriented around regular content and the current top-level settings in typography.scss are rooted in the landing page—i.e., that the exception is the landing page, not the legal page.

Anyway, something to chew on for future updates.